### PR TITLE
Add Asymmetric Tiling

### PR DESCRIPTION
**Pull request by carson-katri**
This adds a new ability to the `seamless` option to specify which axes to apply the circular convolution to. You can use the `seamless_axes` arg to specify which axes to tile:
`brick wall texture --seamless --seamless_axes=x`

The implementation is based on this [asymmetric-tiling](https://github.com/tjm35/asymmetric-tiling-sd-webui) repo.

Here's some sample results:
### Seamless XY

seamless_xy

Screen Shot 2022-10-16 at 10 50 31 PM

### Seamless X

seamless_x

Screen Shot 2022-10-16 at 10 50 00 PM

### Seamless Y

seamless_y

Screen Shot 2022-10-16 at 10 49 26 PM

### CODE REVIEW

**Review by lstein — CHANGES_REQUESTED** [review]
Thanks so much for your contribution. Could you also update the documentation to tell the user how to take advantage of this new feature?

File: ldm/generate.py [hunk 1]
```diff
@@ -348,9 +351,30 @@ def process_image(image,seed):
         width = width or self.width
         height = height or self.height
         
+        def _conv_forward_asymmetric(self, input, weight, bias):
```
Comment (lstein) [inline ldm/generate.py:354]: `generate.py` has gotten very large, and I wonder if it makes sense to move this and the rest of the seamless tiling code into a module named something like `ldm/invoke/seamless.py` ?

Comment (carson-katri) [reply ldm/generate.py]: I moved it there in 99581db, let me know if you want it structured or named differently.

**Comment by carson-katri** [comment]
Documentation has been updated as well in 4079333

**Review by lstein — APPROVED** [review]
LGTM

**Pull request merged**